xformers/components/multi_head_dispatch.py

```diff
@@ -12,7 +12,7 @@
 from torch.nn.init import constant_
 
 from xformers.components.attention import Attention
-from xformers.components.in_proj_container import InProjContainer, InProjParams
```

Comment:
renaming this, I thought it was a little easier to understand what it was doing from the outside